Libraries/StyleSheet/processTextDecorationLine.android.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule processTextDecorationLine
+ */
+'use strict';
+
+const textDecorationLineBitFlags = {
+  'none': 0,
+  'underline': 1 << 0,
+  'line-through': 1 << 1,
+};
+
+// textDecorationLine can be a string with a single enum value, a string with
+// space-separated enum values, or an array of enum values.  Because ReactAndroid
```

Comment:
The array syntax is better IMO since the string syntax is legacy from CSS, which doesn't have arrays. This PR keeps around the old string syntax in case there are apps depending on it.